plugin/storage/es/factory.go

```diff
@@ -204,7 +204,11 @@ func (f *Factory) CreateArchiveSpanReader() (spanstore.Reader, error) {
 	if !f.archiveConfig.Enabled {
 		return nil, nil
 	}
-	sr, err := createSpanReader(f.getArchiveClient, f.archiveConfig, true, f.logger, f.tracer)
+	readAlias := "archive"
```

Comment:
let's use consistent naming everywhere
```suggestion
	readAliasSuffix := "archive"
```